### PR TITLE
DOC-11447 Document new User/System Connections Alert

### PR DESCRIPTION
**Pull request by RayOffiah**
https://issues.couchbase.com/browse/DOC-11447

https://issues.couchbase.com/browse/MB-59666

Test page:

https://rayoffiah.github.io/DOC-11447/server/7.2/manage/manage-settings/configure-alerts.html

### CODE REVIEW

**Comment by malarky** [comment]
There are actually two different alerts - one for User and one for System.  As requested in [DOC-11447](https://issues.couchbase.com/browse/DOC-11447), the docs should provide guidance to contact support if the System connections alert fires.

**Comment by malarky** [comment]
The [REST API](https://docs.couchbase.com/server/current/rest-api/rest-cluster-email-notifications.html) and [CLI](https://docs.couchbase.com/server/current/cli/cbcli/couchbase-cli-setting-alert.html) docs also need updating to add the new alerts.

**Comment by RayOffiah** [comment]
> The [REST API](https://docs.couchbase.com/server/current/rest-api/rest-cluster-email-notifications.html) and [CLI](https://docs.couchbase.com/server/current/cli/cbcli/couchbase-cli-setting-alert.html) docs also need updating to add the new alerts.

@malarky 

Added information on setting the threshold values. I understand that there is no CLI for this stuff at the moment, and the alert is shared by both thresholds

File: modules/manage/pages/manage-settings/configure-alerts.adoc [hunk 1]
```diff
@@ -196,8 +196,19 @@ For information, on establishing change-history size, see xref:rest-api:rest-buc
 | Warns that the Index Service is, on a given node, occupying a percentage of available memory that is below an established threshold, the default for which is `10`.
 | `lowIndexerResidentPerc`
 
+a| [#memcached-alert]
+Memcached threshold exceeded.
```
Comment (BenHuddleston) [inline modules/manage/pages/manage-settings/configure-alerts.adoc]: Probably should be "Memcached connection threshold exceeded"

File: modules/manage/pages/manage-settings/configure-alerts.adoc [hunk 2]
```diff
@@ -196,8 +196,19 @@ For information, on establishing change-history size, see xref:rest-api:rest-buc
...
+| Trigger an alert if the number of `system` or `user` connections used by the data service exceeds a configurable percentage of the available connections{blank}xref:#memcached-alert-foonote[^1^].
+For information on setting the `memcache` alert thresholds, see xref:rest-api:rest-cluster-email-notifications.adoc#setting-memcache-alert-threshold[Setting alerts].
```
Comment (BenHuddleston) [inline modules/manage/pages/manage-settings/configure-alerts.adoc]: "memcache" should perhas be "memcached"?

**Review by BenHuddleston — COMMENTED** [review]
Perhaps we want to recommend that users do not change the system connection threshold, but that's probably a question for @malarky.

**Comment by RayOffiah** [comment]
@malarky 
@BenHuddleston 

Unless there are any objections, I'll merge this later today

**Pull request merged**